services/searchengine/nullengine/null.go

```diff
@@ -0,0 +1,99 @@
+package nullengine
```

Comment:
What do you think about noopengine/dummyengine ?

Reply:
I don't know which one is the best naming, actually, I'm not 100% sure if we need it. I'll think about it.